modules/mono/editor/bindings_generator.cpp

```diff
@@ -522,7 +543,22 @@ void BindingsGenerator::_append_xml_method(StringBuilder &p_xml_output, const Ty
 				p_xml_output.append(p_target_itype->proxy_name);
 				p_xml_output.append(".");
 				p_xml_output.append(target_imethod->proxy_name);
-				p_xml_output.append("\"/>");
+				p_xml_output.append("(");
+				const ArgumentInterface &first = target_imethod->arguments.front()->get();
+				for (const ArgumentInterface &iarg : target_imethod->arguments) {
```

Comment:
You could also use a boolean like https://github.com/godotengine/godot/pull/50828.

Reply:
Oh, yeah, that'd be way easier lmao